build.gradle

```diff
@@ -26,6 +26,16 @@ plugins {
 
 	// https://spotbugs.readthedocs.io/en/latest/gradle.html
 	id "com.github.spotbugs" version "6.0.14"
+
+	// https://plugins.gradle.org/plugin/name.remal.sonarlint
+	// https://github.com/remal-gradle-plugins/sonarlint
+	// https://plugins.gradle.org/search?term=sonarlint
+	// https://remal.gitlab.io/gradle-plugins/
+	// id "name.remal.sonarlint" version "4.2.1"
+
+	// or
+	// https://github.com/Lucas3oo/sonarlint-gradle-plugin
+	id "se.solrike.sonarlint" version "2.0.0"
```

Comment:
Vulnerabilities here should be irrelevant.

They appear only for build tools, specifically for sonarlint, no code gets used in the lib.

for solrike, it's using dependency   
```
implementation('io.github.furstenheim:copy_down:1.1')
```
check https://github.com/Lucas3oo/sonarlint-gradle-plugin/blob/main/build.gradle 

which imports
```
implementation 'org.jsoup:jsoup:1.15.2'
```

check https://github.com/furstenheim/copy-down/blob/master/build.gradle 

which has cve https://mvnrepository.com/artifact/org.jsoup/jsoup/1.15.2 

Should we let this pass as a build tool?

Reply:
vuln: 
![image](https://github.com/gipo999/test-spi/assets/91525512/7b154dc9-38aa-4509-9b6b-466fd418cb20)

from https://github.com/gipo999/test-spi/actions/runs/9061509492/job/24893404724?pr=44

Reply:
Yes, we can let this pass (it's only a tool and not a final build dependency)